ignite/metrics/ssim.py

```diff
@@ -121,16 +121,16 @@ def update(self, output: Sequence[torch.Tensor]) -> None:
         if len(self._kernel.shape) < 4:
             self._kernel = self._kernel.expand(channel, 1, -1, -1).to(device=y_pred.device)
 
-        mu_pred = F.conv2d(y_pred, self._kernel, groups=channel)
-        mu_target = F.conv2d(y, self._kernel, groups=channel)
+        input_list = [y_pred, y, y_pred * y_pred, y * y, y_pred * y]
+        output_list = [F.conv2d(x, self._kernel, groups=channel) for x in input_list]
```

Comment:
Here you still have 5 conv ops. IMO, I prefer the previous code as more clean. Idea was to have a single conv op on concatenated input...

Reply:
Ahhh,  I get it.

Reply:
also we need to use `skimage.measure` for py3.5 since the only tagged version in req.txt has ssim metrics and support py3.5

Reply:
Maybe, we can just add 
```python
try:
    from skimage.metrics import structural_similarity as ski_ssim
except ImportError:
    from skimage.measure ...
```

Reply:
ssim implementation of skimage between different python version is still the same.